src/Node.ts

```diff
@@ -3102,7 +3102,7 @@ addGetterSetter(Node, 'offsetY', 0, getNumberValidator());
  * node.offsetY(3);
  */
 
-addGetterSetter(Node, 'dragDistance', null, getNumberValidator());
+addGetterSetter(Node, 'dragDistance', undefined, getNumberValidator());
```

Comment:
This is a subtle API change.

Perhaps we should widen the type of `dragDistance` in `NodeConfig` to `dragDistance?: number | null;`?

Reply:
The drag distance must be always number. So `undefined` should fallback to default value. I am not really sure what it was `null` here.